compiler/rustc_codegen_cranelift/src/lib.rs

```diff
@@ -1,8 +1,10 @@
-#![feature(rustc_private)]
 // Note: please avoid adding other feature gates where possible
+#![feature(rustc_private)]
+// tidy-alphabetical-start
 #![warn(rust_2018_idioms)]
-#![warn(unused_lifetimes)]
 #![warn(unreachable_pub)]
+#![warn(unused_lifetimes)]
+// tidy-alphabetical-end
```

Comment:
Could you please skip this for cg_clif? There is nothing in place in https://github.com/bjorn3/rustc_codegen_cranelift to enforce this and because of this it will only cause issues when syncing between that repo and this repo.

Reply:
Same applies to rust-analyzer, I'd say this should not be done for subtrees in general.